### DISCUSSION TITLE
BibTex comments --> Zotero Notes?

### DISCUSSION THREAD

**Discussion by notuntoward** [retorquere/zotero-better-bibtex#2788] [Q&A]
I'd like to import a bibtex file into zotero, but I don't know how to import the bibtex comments field into the corresponding zotero note for each entry.  Is there a way?  

Google Gemni tells me what I want to hear, that it's possible to 'In the Better BibTeX settings, locate the "Import" tab and enable the "Import BibTeX comments" option.'  But shocker, I can't find such a thing.

**Comment by retorquere** [comment]
There's no setting, just import the file with the `comment =` fields.

**Reply by notuntoward** [reply]
Ah, thanks!

Does this part of import have actually have anything to do with Better BibTex, or is it purely built-in Zotero functionality?

I'm asking because the bibtex comment field is in markdown, but arrives in the imported zotero note with all the newlines removed, eliminating the markdown structure.

If this isn't handled in Better Bibtex, sorry for asking a Zotero question here.

**Reply by retorquere** [reply]
If you have bbt installed, and you've not changed the import settings, BBT handles bibtex import.

If you have a sample for me I can have a look, but BBT doesn't grok markdown, so options may be limited.

**Reply by notuntoward** [reply]
Thanks for taking a look.  I've attached a sample entry, and here's the comment as displayed in a zotero note.

![image](https://github.com/retorquere/zotero-better-bibtex/assets/3851891/aed7ec31-a212-4011-825f-5e053dec9ec2)
[a_dot_bib_file.txt](https://github.com/retorquere/zotero-better-bibtex/files/14274874/a_dot_bib_file.txt)

**Reply by retorquere** [reply]
There's an error in my parser. I'll get out a fix.

**Reply by retorquere** [reply]
Can you try the version from #2789? It should do better.

**Reply by notuntoward** [reply]
Thanks, the double newline processing helps a lot.  

Depending upon how far you want to go... it would be even better if it recognized a bit more markdown, especially bullets (a new line with the 1st nonwhitespace character being '-').  Here's how the note looks now:

![image](https://github.com/retorquere/zotero-better-bibtex/assets/3851891/618339b9-2f1e-4cf1-8c42-386a6c5ac91c)

**Reply by retorquere** [reply]
There's no reliable way for me to detect that content is markdown rather than latex.
